Tiltfile

```diff
@@ -34,6 +34,8 @@ def launch_go_svc(name, dirname="", flags="", auto_init=True):
         flags if flags else ""
     )
     env = {
+        'SERVICE_NAME': name,
+        'OTEL_SERVICE_NAME': name,

```

Comment:
The distro only looks for and configures itself using`OTEL_SERVICE_NAME`. The spec originally used `SERVICE_N AME` but now recommends the newer one instead.